foreman/data_refinery_foreman/foreman/management/commands/create_compendia.py

```diff
@@ -58,33 +29,130 @@ def add_arguments(self, parser):
             help=("Specify SVD algorithm applied during imputation ARPACK, RANDOMIZED or NONE to skip."))
 
     def handle(self, *args, **options):
-        """Create a compendium for one or more organisms.
+        """Create a compendium for one or more organisms."""
+        target_organisms = self._get_target_organisms(options)
+        grouped_organisms = group_organisms_by_biggest_platform(target_organisms)
+
+        quant_sf_only = options["quant_sf_only"] is True
```

Comment:
This is now never an option for this command.

Reply:
good catch! just removed it